bitnami/influxdb/README.md

```diff
@@ -0,0 +1,348 @@
+| `ingress.secrets[0].key`                    | TLS Secret Key                                                                                                                                                                            | `nil`                                                   |
+| `networkPolicy.enabled`                     | Enable NetworkPolicy                                                                                                                                                                      | `false`                                                 |
+| `networkPolicy.allowExternal`               | Don't require client label for connections                                                                                                                                                | `true`                                                  |
+| `persistence.enabled`                       | Enable data persistence                                                                                                                                                                   | `true`                                                  |
+| `persistence.existingClaim`                 | Use a existing PVC which must be created manually before bound                                                                                                                            | `nil`                                                   |
+| `persistence.storageClass`                  | Specify the `storageClass` used to provision the volume                                                                                                                                   | `nil`                                                   |
+| `persistence.accessMode`                    | Access mode of data volume                                                                                                                                                                | `ReadWriteOnce`                                         |
+| `persistence.size`                          | Size of data volume                                                                                                                                                                       | `8Gi`                                                   |
+| `volumePermissions.enabled`                 | Enable init container that changes the owner and group of the persistent volume mountpoint to `runAsUser:fsGroup`                                                                         | `false`                                                 |
+| `volumePermissions.image.registry`          | Init container volume-permissions image registry                                                                                                                                          | `docker.io`                                             |
```

Comment:
should volume-permission have backticks? `volume-permissions`

Reply:
Why? We're using camelCase everywhere

Reply:
Why?